starter-code/public/scripts/controllers/routes.js

```diff
@@ -12,5 +12,5 @@ page('/author', '/');
 page('/author/:authorName', app.articleController.loadByAuthor, app.articleController.index);
 page('/category/:categoryName', app.articleController.loadByCategory, app.articleController.index);
 
-// COMMENT: What is this function doing?
+// : What is this function doing? (RESPONSE: The empty page function extis the middleware by telling it to stop looking for nexts.)
```

Comment:
Not quite. The `page()` call run once, as part of initializing the page. Middleware are processed for every new client-side "page" request.

So what does `page()` actually do?